src/pages/ReportDetailsPage.tsx

```diff
@@ -44,6 +44,7 @@ import {getPersonalDetailsForAccountIDs} from '@libs/OptionsListUtils';
 import Parser from '@libs/Parser';
 import {getConnectedIntegration, isPolicyAdmin as isPolicyAdminUtil, isPolicyEmployee as isPolicyEmployeeUtil, shouldShowPolicy} from '@libs/PolicyUtils';
 import {getOneTransactionThreadReportID, getOriginalMessage, getTrackExpenseActionableWhisper, isDeletedAction, isMoneyRequestAction, isTrackExpenseAction} from '@libs/ReportActionsUtils';
+import {isDeleteAction} from '@libs/ReportSecondaryActionUtils';
```

Comment:
The changes in this file don't seem correct. In fact, we should remove the delete option from the details page since it'll now live in the `More` button